index.js

```diff
@@ -283,13 +284,15 @@ module.exports = class Hyperdrive extends EventEmitter {
 
   // atm always recursive, but we should add some depth thing to it
   list (folder, { recursive = true } = {}) {
+    folder = folder || '/'
     if (folder.endsWith('/')) folder = folder.slice(0, -1)
     if (recursive === false) return shallowReadStream(this.files, folder, false)
     // '0' is binary +1 of /
     return folder ? this.entries({ gt: folder + '/', lt: folder + '0' }) : this.entries()
   }
 
   readdir (folder) {
+    folder = folder || '/'
```

Comment:
if (!folder) folder = '/'

everywhere :)

Reply:
or just folder = '/' in the args

Reply:
Would it be better the condition to cover null values?

Reply:
i'd just use the default arg, we rely on that elsewhere as well, so need need to be anymore defensive than that